docs/deep-linking.md

```diff
@@ -141,19 +139,26 @@ To test the URI on a real device, open Safari and type `mychat://chat/Eric`.
 
 To configure the external linking in Android, you can create a new intent in the manifest.
 
-In `SimpleApp/android/app/src/main/AndroidManifest.xml`, add the new `intent-filter` inside the `MainActivity` entry with a `VIEW` type action:
+In `SimpleApp/android/app/src/main/AndroidManifest.xml`, do these followings adjustments:
+1. Set `launchMode` of `MainActivity` to `singleTask` in order to receive intent on existing `MainActivity`. It is useful if you want to perform navigation using deep link you have been registered - [details](http://developer.android.com/training/app-indexing/deep-linking.html#adding-filters)
+2. Add the new `intent-filter` inside the `MainActivity` entry with a `VIEW` type action:
 
 ```
-<intent-filter>
-    <action android:name="android.intent.action.MAIN" />
-    <category android:name="android.intent.category.LAUNCHER" />
-</intent-filter>
-<intent-filter>
-    <action android:name="android.intent.action.VIEW" />
-    <category android:name="android.intent.category.DEFAULT" />
-    <category android:name="android.intent.category.BROWSABLE" />
-    <data android:scheme="mychat" android:host="mychat" />            
-</intent-filter>
+<activity
+    android:name=".MainActivity"
+    android:launchMode="singleTask"
+    ...>
```

Comment:
ideally code should be copy+pasteable from docs, so we should remove the `...` here

Reply:
Updated